src/microbe_stage/editor/MicrobeEditorGUI.cs

```diff
@@ -915,6 +922,32 @@ public void UpdateAutoEvoResults(string results, string external)
         externalEffectsLabel.Text = external;
     }
 
+    public void ShowOrganelleMenu(OrganelleTemplate selectedOrganelle)
+    {
+        organelleMenu.SelectedOrganelle = selectedOrganelle;
+        organelleMenu.ShowPopup = true;
+
+        // Disable delete option for nucleus or the last organelle.
```

Comment:
This comment is no longer true. There now seems to be a bug that the nucleus can be deleted.
Also move *must* be prevented if only one organelle is in the species as that will permanently break the ATP balance bar.